DateRangeBox: add new API description (#5026)
DevExpress/devextreme-documentation: api-reference/10 UI Components/DateBoxBase/1 Configuration/dateOutOfRangeMessage.md

---
id: DateBoxBase.Options.dateOutOfRangeMessage
type: String
default: 'Value is out of range'
---
---
##### shortDescription

> Specifies the message displayed if the specified date is later than the [max]({basewidgetpath}/Configuration/#max) value or earlier than the [min]({basewidgetpath}/Configuration/#min) value.

Reviewer: ```suggestion
Specifies the message displayed if the date is after the [max]({basewidgetpath}/Configuration/#max) value or before the [min]({basewidgetpath}/Configuration/#min) value.
```
